docs/raredisease_map.md

```diff
@@ -0,0 +1,81 @@
+| gens_generatedata, cov                            | False       | gens, coverage, bed                     | scout        |
+| gens_generatedata, cov_index                      | False       | gens, coverage, bed-index               | scout        |
+| call_mobile_elements, call_mobile_elements        | False       | gens, coverage, bed-index               | scout        |
+| call_mobile_elements, call_mobile_elements_index  | False       | gens, coverage, bed-index               | scout        |
+| annotate_mobile_elements, clinical                | False       | mobile-elements, clinical, vcf          | scout        |
+| annotate_mobile_elements, clinical_index          | False       | mobile-elements, clinical, vcf_index    | scout        |
+| annotate_mobile_elements, research                | False       | mobile-elements, research, vcf          | scout        |
+| annotate_mobile_elements, research_index          | False       | mobile-elements, research, vcf_index    | scout        |
+| call_snv, call_snv                                | True        | vcf_snv                                 | genotype     |
+| call_snv, call_snv_index                          | False       | vcf_snv_index                           | scout        |
+| call_snv, call_snv_mt                             | False       | vcf_snv, mitochondria                   | scout        |
+| call_snv, call_snv_mt_index                       | False       | vcf_snv_index, mitochondria             | scout        |
+| call_sv, call_sv                                  | False       | vcf_sv                                  | scout        |
+| call_sv, call_sv_index                            | False       | vcf_sv_index                            | scout        |
+| call_sv, call_sv_mt                               | False       | vcf_sv, mitochondria                    | scout        |
+| call_sv, call_sv_mt_index                         | False       | vcf_sv_index, mitochondria              | scout        |
+| call_sv_mt, call_sv_mt_del                        | False       | mitodel, mitochondria                   | scout        |
+| call_sv_mt, eklipse_del                           | False       | eklipse_del, csv, mitochondria          | scout        |
+| call_sv_mt, eklipse_png                           | False       | eklipse_png, mitochondria               | scout        |
+| call_sv_mt, eklipse_genes                         | False       | eklipse_genes, mitochondria             | scout        |
+| annotate_sv, svdbquery                            | False       | svdbquery, vcf                          | scout        |
+| annotate_sv, svdbquery_index                      | False       | svdbquery, vcf_index                    | scout        |
+| rank_and_filter, snv_clinical                     | True        | vcf-snv-clinical                        | scout        |
```

Comment:
clinical and research should be separated tags

Reply:
Same here:https://github.com/Clinical-Genomics/hermes/pull/109#discussion_r1598319025